frontend/src/metabase/collections/components/CollectionHeader/CollectionUpload.tsx

```diff
@@ -0,0 +1,55 @@
+import React, { useState, useEffect } from "react";
+
+import type { Collection, CollectionId } from "metabase-types/api";
+import { color } from "metabase/lib/colors";
+
+import { CollectionHeaderButton } from "./CollectionHeader.styled";
+import { UploadInput } from "./CollectionUpload.styled";
+
+const CLEAR_ERROR_TIMEOUT = 3000;
+
+export default function ColllectionUpload({
+  collection,
+  onUpload,
+}: {
+  collection: Collection;
+  onUpload: (file: File, collectionId: CollectionId) => void;
+}) {
+  const [error, setError] = useState("");
```

Comment:
Where `error` is being set?

Reply:
ah good catch. there's a bunch of code here that became defunct when the notification got put in.